src/pipe_log.rs

```diff
@@ -210,86 +270,46 @@ impl LogManager {
             self.active_log_capacity += alloc_size;
         }
 
-        if sync || self.reach_sync_limit() {
+        if *sync || self.reach_sync_limit() {
             self.last_sync_size = self.active_log_size;
-            sync = true;
+            *sync = true;
```

Comment:
It is unnecessary because OS will flush the page cache to disk automatic.  So we do not need to check the size of data in os buffer.

Reply:
Generally `bytes_per_sync` is less than the OS dirty page limit. So It's still meaningful.